addons/EasyNuke/EasyNuke.lua

```diff
@@ -0,0 +1,207 @@
+			windower.send_command("@input /echo Element Mode is now: "..string.ucfirst(current_element))
+		elseif other_modes:contains(arg) then
+			current_element = arg
+			windower.send_command("@input /echo Element Mode is now: "..string.ucfirst(current_element))
+		else windower.send_command("@input /echo Invalid element")
+		end
+	end
+	
+	if command == "cycle" then
+		arg = string.lower(arg)
+		if arg == nil then
+			find_index()
+			elements_index = elements_index + 1
+			if elements_index > 8 then elements_index = 1 end
+			current_element = tostring(elements[elements_index])
+			windower.send_command("@input /echo Element Mode is now: "..string.ucfirst(current_element))
+		elseif arg == "back" then
+			find_index()
+			elements_index = elements_index - 1
+			if elements_index < 1 then elements_index = 8 end
+			current_element = elements[elements_index]
+			windower.send_command("@input /echo Element Mode is now: "..string.ucfirst(current_element))
+		elseif arg == "dark" then
+			find_index()
+			elements_index = elements_index + 1
+			if elements_index < 5 then elements_index = 5 end
+			if elements_index > 8 then elements_index = 5 end
+			current_element = elements[elements_index]
+			windower.send_command("@input /echo Element Mode is now: "..string.ucfirst(current_element))
+		elseif arg == "light" then
+			find_index()
+			elements_index = elements_index + 1
+			if elements_index > 4 then elements_index = 1 end
+			current_element = elements[elements_index]
+			windower.send_command("@input /echo Element Mode is now: "..string.ucfirst(current_element))
+		elseif arg == "fusion" then
+			if current_element ~= "fire" and current_element ~= "light" then current_element = "fire"
+			elseif current_element == "fire" then current_element = "light"
+			elseif current_element == "light" then current_element = "fire"
+			end
+		elseif arg == "distortion" or arg == "dist" or arg == "d" then
+			if current_element ~= "ice" and current_element ~= "water" then current_element = "ice"
+			elseif current_element == "ice" then current_element = "water"
+			elseif current_element == "water" then current_element = "ice"
+			end
+		elseif arg == "gravitation" or arg == "grav" or arg == "g" then
+			if current_element ~= "earth" and current_element ~= "dark" then current_element = "earth"
+			elseif current_element == "earth" then current_element = "dark"
+			elseif current_element == "dark" then current_element = "earth"
+			end
+		elseif arg == "fragmentation" or arg == "frag" or arg == "f" then
+			if current_element ~= "thunder" and current_element ~= "wind" then current_element = "thunder"
+			elseif current_element == "thunder" then current_element = "wind"
+			elseif current_element == "wind" then current_element = "thunder"
+			end
+		end
+	end
+		
+	if command == "showcurrent" then
+		windower.send_command("@input /echo ----- Element Mode: "..string.ucfirst(current_element).." --- Target Mode: < "..target_mode.." > -----")
```

Comment:
`windower.chat.input` saves you from having to preface every command with `@input ` and will hopefully soon also be faster than `windower.send_command`.

Reply:
Is there any benefit to changing this?  Does it actually function differently or is this an alternative method to do the exact same thing?

Reply:
```lua
windower.add_to_chat(206, "----- Element Mode: "..string.ucfirst(current_element).." --- Target Mode: < "..target_mode.." > -----")
```
This is faster, less code, and lets you change the color of the text (the `206`) easily.

For the cases where you're trying to send a `/command` to the game,
```lua
windower.chat.input('/command')
```
This is much simpler, and faster, since it skips the Windower command handler.